### PR TITLE
F368-grey out and add event code to name field when creating private queue

### PR DESCRIPTION
**Pull request by JohnnyPersichilli**

### CODE REVIEW

File: CranberryQueue/Map/MapViewController.swift [hunk 1]
```diff
@@ -349,6 +354,18 @@ class MapViewController: UIViewController, UITextFieldDelegate, MapControllerDel
         }
     }
     
+    @objc func createPrivateButtonTapped(){
+        let remote = (UIApplication.shared.delegate as? AppDelegate)?.appRemote
+        remote?.playerAPI!.resume({ (response, error) in
```
Comment (rolflocher) [inline CranberryQueue/Map/MapViewController.swift]: Why do you need to resume here?

File: CranberryQueue/Map/createQueueForm.swift [hunk 2]
```diff
@@ -36,15 +39,48 @@ class createQueueForm: UIView {
...
+        createPrivateButton.isHidden = true
     }
     
     @IBAction func switchChanged(_ sender: Any) {
         if scopeSwitch.isOn {
+            createPrivateButton.isHidden = true
+            
+            privateCode = nil
+            queueNameTextField.text = ""
+            queueNameTextField.textColor = UIColor.black
```
Comment (rolflocher) [inline CranberryQueue/Map/createQueueForm.swift]: I dont understand where this ticket came from. I know I didnt agree to any of these changes

Comment (rolflocher) [reply CranberryQueue/Map/createQueueForm.swift]: You shouldnt make a ticket and a branch for it without telling anybody. I know I do that sometimes, but only in group meetings where we are solving bugs together etc. I can understand bringing a bug into the sprint on the fly, but you should ask everyone about a new feature

File: CranberryQueue/Map/MapViewController.swift [hunk 3]
```diff
@@ -349,32 +354,50 @@ class MapViewController: UIViewController, UITextFieldDelegate, MapControllerDel
...
         if textField.text == nil || textField.text == "" {
+            let alert = UIAlertController(title: "Name must not be empty", message: "", preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
+            self.present(alert, animated: true)
             return false
         }
     
         /// determine which textfield called the delegate
         if textField == createQueueForm.queueNameTextField {
-            //resume music
```
Comment (JohnnyPersichilli) [inline CranberryQueue/Map/MapViewController.swift]: logic moved to other function to be reused on join click and not just keyboard return

File: CranberryQueue/Map/MapViewController.swift [hunk 4]
```diff
@@ -393,25 +416,6 @@ class MapViewController: UIViewController, UITextFieldDelegate, MapControllerDel
         delegate.seshDelegate = self
     }
     
-    
-    // Takes the current timestamp in decimal and returns a short string of base n
-    func eventCodeFromTimestamp() -> String {
```
Comment (JohnnyPersichilli) [inline CranberryQueue/Map/MapViewController.swift]: moved to create queue form

File: CranberryQueue/Map/createQueueForm.swift [hunk 5]
```diff
@@ -34,17 +38,58 @@ class createQueueForm: UIView {
...
+        modalView.layer.borderColor = UIColor.black.cgColor
+        
+        queueNameTextField.borderStyle = .roundedRect
         queueNameTextField.returnKeyType = .join
     }
     
     @IBAction func switchChanged(_ sender: Any) {
         if scopeSwitch.isOn {
-            scopeLabel.text = "Public"
+            setPublicToggleUI()
```
Comment (JohnnyPersichilli) [inline CranberryQueue/Map/createQueueForm.swift]: made this a function to be reused when closing the modal and for ease of use.  This solved a bug we previously hadn't noticed where after closing the modal, the toggle would switch to on but not trigger the switchedChanged function resulting in the toggle showing the public state but with the text and other attributes in the private state

File: CranberryQueue/Map/createQueueForm.swift [hunk 6]
```diff
@@ -34,17 +38,58 @@ class createQueueForm: UIView {
...
+            queueNameTextField.becomeFirstResponder()
```
Comment (JohnnyPersichilli) [inline CranberryQueue/Map/createQueueForm.swift]: needed to keep this out of the function to ensure that the keyboard would not open in the queue screen as a result of closing the create modal

**Pull request closed without merging**